Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG GO_VERSION=1.18
+ARG ALPINE_VERSION=3.16
```

Comment:
These will need a change

Reply:
Fixed, set go version according to go.mod and up alpine version